src/url-setters.cpp

```diff
@@ -153,7 +153,8 @@ namespace ada {
   bool url::set_pathname(const std::string_view input) {
     if (has_opaque_path) { return false; }
     path = "";
-    return parse_path(input);
+    parse_path(input);
+    return true;
```

Comment:
I think this is a breaking change. Even though you give a false input, it always returns `true`

Reply:
I believe our test coverage does not cover this scenario.

Reply:
parse_path(input) would always return `true` so `return parse_path(input);` was always returning true.